app/controllers/affiliates_controller.rb

```diff
@@ -0,0 +1,69 @@
+class AffiliatesController < ApplicationController
+  before_action :set_affiliate, only: [:show, :edit, :update, :destroy]
+
+  # GET /affiliates
+  def index
+    @affiliates = Affiliate.all
+  end
+
+  # GET /affiliates/1
+  # TODO: should all coordinators, students, and tutors within this affiliate
+  def show
+    @affiliate = Affiliate.find(params[:id])
```

Comment:
You can still define other stuff! `set_affiliate` will still work. take this out and test it. :)